packages/eslint-config-qlean-react-native/README.md

```diff
@@ -0,0 +1,53 @@
+# eslint-config-qlean-react-native
+
+This package provides Qlean's `.eslintrc` as an extensible shared config for linting React Native codebase.
```

Comment:
1. It's airbnb config copypaste
2. It's not true, cuz we extending airbnb config

Reply:
@alextewpin It is true. We use our config as extensible shared config and it doesn't matter that we extends Airbnb or whatever. Any suggestions?

Reply:
I think it's fine to just cut this paragraph and leave this kind of blablabla stuff to main `README`, `eslint-config-qlean-react-native` is self-explanatory enough.